db/queries/categories.js

```diff
@@ -0,0 +1,61 @@
+const db = require('../connection');
+/* Fetch Categories and count of all items in that particular*/
+
+const getCategory = (categoryId) => {
```

Comment:
I believe we needed all categories as a list, yes? This query returns one category if I'm not mistaken.

Reply:
yeah, I have implemented another query for all categories now